scripts/generators/generateFonts.ts

```diff
@@ -106,13 +107,22 @@ const buildFontFile = async (type: FontType, fontResult: GenerateFontResult, inp
  * @returns {string} Dart definition line used in body of `Icons.dart`.
  */
 function getDartIconDefinition(iconName: string, unicode: string, type: FontType | undefined): string {
+  const name_link = `${iconName}_${type === undefined ? "round" : type}`;
+  const readableName = iconName
+    .split("_")
+    .map((e) => e.capitalize())
+    .join(" ");
+
   if (type == undefined) {
     iconName = iconName.toSnakeCase();
   } else {
     iconName = getIconFileName(iconName, type);
   }
 
-  return `  static const IconData ${iconName} = IconData(0x${unicode}, fontFamily: family${
+  return `  /// <i> <img width='48' src="${GITHUB_URL}${name_link}.png"></br>${readableName} icon ${
```

Comment:
For clarity can we abstract this away into a function like getIconPreview or something?

Reply:
done